projects/developer/src/app/common/components/live-range-selector/component.scss

```diff
@@ -0,0 +1,77 @@
+.live-range-selector {
+    align-items: flex-end;
+
+    div {
+        margin-right: 20px;
```

Comment:
Does this stack and cause 40px of margin right of the `.live-range-selector`, 20 for `<div class="live-range-selector__date-filter-dropdown">` and another 20 for the `<div class="ui-inputgroup">` ?

Reply:
The only place this component is used right now is the dashboard, which I'm guessing will need all sorts of layout fixes at some point. This component seems to do fine above about 385px window width.